File: 2023-2024/sourcecode/mod3/lab11/RoomBasicsJava/app/src/main/java/com/example/roombasicsjava/ui/UserViewHolder.java
package com.example.roombasicsjava.ui;

import androidx.recyclerview.widget.RecyclerView;

import com.example.roombasicsjava.data.User;
import com.example.roombasicsjava.databinding.RvItemBinding;

public class UserViewHolder extends RecyclerView.ViewHolder {
    private RvItemBinding binding;

    public UserViewHolder(RvItemBinding binding) {
        super(binding.getRoot());
        this.binding = binding;
    }

    public void bind(User item) {
        binding.textView.setText(item.getFirstName() + " " + item.getLastName());
    }
}

File: 2023-2024/sourcecode/mod2/lab9/RoomBasicsKotlin/app/src/main/java/com/example/roombasicskotlin/ui/UserViewHolder.kt
package com.example.roombasicskotlin.ui

import androidx.recyclerview.widget.RecyclerView
import com.example.roombasicskotlin.data.User
import com.example.roombasicskotlin.databinding.RvItemBinding

class UserViewHolder(private val binding: RvItemBinding) : RecyclerView.ViewHolder(binding.root) {
    fun bind(item: User) {
        binding.textView.text = "${item.firstName} ${item.lastName}"
    }
}